package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keep-network/sortition-pools",
-  "version": "1.1.2",
+  "version": "1.2.0-pre.0",
```

Comment:
The most recent version published to NPM is `1.2.0-pre.4`. Should it be reflected here?

Reply:
`pre` versions for publication get auto-bumped by the publication script. As we're not updating the version in `package.json` in the repo I'm not sure if we want to update it here.